.github/workflows/spell.yaml

```diff
@@ -0,0 +1,26 @@
+name: Spelling Check
+
+on: [push, pull_request]
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - uses: actions/cache@v2
+      name: Configure pip caching
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-spellcheck
+        restore-keys: |
+          ${{ runner.os }}-pip-spellcheck
```

Comment:
Trying executing `pip install codespell` inside my docker is fast. So why use actions/cache here?

Reply:
copy from somewhere